src/client-side-metrics/exporter.ts

```diff
@@ -0,0 +1,460 @@
+    for (const metric of scopeMetrics.metrics) {
+      const metricName = metric.descriptor.name;
+      if (isCounterMetric(metric)) {
+        for (const dataPoint of metric.dataPoints) {
+          // Extract attributes to labels based on their intended target (resource or metric)
+          const allAttributes = dataPoint.attributes;
+          const streaming = allAttributes.streamingOperation;
+          const metricLabels = Object.assign(
+            {
+              app_profile: allAttributes.appProfileId,
+              client_name: allAttributes.clientName,
+              method: allAttributes.methodName,
+              status:
+                (
+                  allAttributes as OnAttemptAttribute
+                ).attemptStatus?.toString() ??
+                (
+                  allAttributes as OnOperationAttribute
+                ).finalOperationStatus?.toString(),
+              client_uid: allAttributes.clientUid,
+            },
+            streaming ? {streaming} : null
+          );
+          const timeSeries = {
+            metric: {
+              type: metricName,
+              labels: metricLabels,
+            },
+            resource: {
+              type: exportArgs.resource._syncAttributes[
+                'monitored_resource.type'
+              ],
+              labels: resourceLabels,
+            },
+            valueType: 'INT64',
+            points: [
+              {
+                interval: {
+                  endTime: {
+                    seconds: dataPoint.endTime[0],
+                  },
+                  startTime: {
+                    seconds: dataPoint.startTime[0],
+                  },
+                },
+                value: {
+                  int64Value: dataPoint.value,
+                },
+              },
+            ],
+          };
+          timeSeriesArray.push(timeSeries);
+        }
+      } else {
+        for (const dataPoint of metric.dataPoints) {
+          // Extract attributes to labels based on their intended target (resource or metric)
+          const allAttributes = dataPoint.attributes;
+          const streaming = allAttributes.streamingOperation;
+          const metricLabels = Object.assign(
+            {
+              app_profile: allAttributes.appProfileId,
+              client_name: allAttributes.clientName,
+              method: allAttributes.methodName,
+              status:
+                (
+                  allAttributes as OnAttemptAttribute
+                ).attemptStatus?.toString() ??
+                (
+                  allAttributes as OnOperationAttribute
+                ).finalOperationStatus?.toString(),
+              client_uid: allAttributes.clientUid,
+            },
+            streaming ? {streaming} : null
+          );
+          const timeSeries = {
+            metric: {
+              type: metricName,
+              labels: metricLabels,
+            },
+            resource: {
```

Comment:
Note: This is where the monitored resource is provided.

Reply:
![image](https://github.com/user-attachments/assets/9c9b396a-e00c-4c47-ab45-826745d504d4)